requirements/testing.in

```diff
@@ -2,6 +2,8 @@ coverage[toml]
 pytest
 pytest-mock
 pytest-pretty
+pytest-tmp-files
+parametrize-from-file
```

Comment:
I don't think we need these extra dependencies, it should be a few lines of code to do this manually, either with strings in the python code, or reading files from a new directory in `tests`.

Reply:
you can probably use python files with a multiline string at the end of the for expected stdout.

Reply:
I rewrote the tests using vanilla python.